### PR TITLE
feat: update readme with instructions and info about the project

### PR DESCRIPTION
**Pull request by tdejager**
Update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,103 @@
...
+
+### Configuration
+#### Config.yaml (what recipes to build) ⚙️
+The `config.yaml` file contains the configuration for the project. You can add `remote` or `local` recipes, that are local to the project or remote repositories.
+The config also contains the rattler-build version that its using, this way we can depend on unreleased versions. This project automatically sets this up for you.
+
+### SQLite Database (stores the build information) 📕
+This project uses a Sqlite database to store the build information. The database is created automatically when the project is setup.
+When running locally a local version of the database is created, this will ensure that you have a clean database to work with.
+You can also use the `--in-memory-sql` flag to use an in-memory database, which is useful for testing.
```
Comment (nichmor) [inline README.md:59]: let's add an example:
`pixi run repror --in-memory-sql build-recipe boltons` because sometimes for me is confusing these global options

File: README.md [hunk 2]
```diff
@@ -7,7 +7,103 @@
 [chat-badge]: https://img.shields.io/discord/1082332781146800168.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2&style=flat-square
 [chat-url]: https://discord.gg/kKV8ZxyzY4
 
+# 🏁🏁 Build Status 🏁🏁
+You can view the current build status for the different packages [here](https://prefix-dev.github.io/reproducible-builds/).
 
+## Introduction
+This project aims to see if we can create reproducible builds for a variety of software packages.
+The project is based on definitions of the [Reproducible Builds](https://reproducible-builds.org/) project.
```
Comment (nichmor) [inline README.md:15]: wdyt about adding a tl;dr about what is for us a reproducible build?

**Review by nichmor — APPROVED** [review]
Nice! 🔥

File: README.md (hunk 2 above)
Comment (tdejager) [reply README.md:15]: Good one what do you suggest?

Comment (nichmor) [reply README.md:15]: Something along the lines:

tl;dr
We validate that `rattler-build` is a reproducible build tool by ensuring that it can recreate identical packages given the same recipe and build environment. We verify this by comparing the `SHA256` hash of the original build and the rebuild. If the hashes differ, we inspect the packages with `diffoscope` to identify the differences. Common issues affecting reproducibility include unsorted files, differing archive timestamps, and reset script permissions during the archiving step.

Comment (tdejager) [reply README.md:15]: Ok I've added that and a bit extra!

**Comment by tdejager** [comment]
Think this should be good to go now :)
